glpi-official/services/web/Dockerfile

```diff
@@ -0,0 +1,118 @@
+  && docker-php-ext-install ldap \
+  \
+  # Install memcached PHP extension.
+  && apt-get install --assume-yes --no-install-recommends --quiet libmemcached-dev \
+  && pecl install memcached \
+  && docker-php-ext-enable memcached \
+  \
+  # Install mysqli PHP extension.
+  && docker-php-ext-install mysqli \
+  \
+  # Install pcntl PHP extension (required for composer-require-checker).
+  && docker-php-ext-install pcntl \
+  \
+  # Install redis PHP extension.
+  && pecl install redis \
+  && docker-php-ext-enable redis \
+  \
+  # Install soap PHP extension (required for some plugins).
+  && apt-get install --assume-yes --no-install-recommends --quiet libxml2-dev \
+  && docker-php-ext-install soap \
+  \
+  # Install XMLRPC PHP extension.
+  # For PHP 8+, install from Github (extension should be available on PECL but is not)
+  && apt-get install --assume-yes --no-install-recommends --quiet libxml2-dev \
+  && mkdir -p /tmp/xmlrpc \
+    && curl -LsfS https://github.com/php/pecl-networking-xmlrpc/archive/0f782ffe52cebd0a65356427b7ab72d48b72d20c/xmlrpc-0f782ff.tar.gz | tar xvz -C "/tmp/xmlrpc" --strip 1 \
+    && docker-php-ext-configure /tmp/xmlrpc --with-xmlrpc \
+    && docker-php-ext-install /tmp/xmlrpc \
+    && rm -rf /tmp/xmlrpc \
+  \
+  # Install zip PHP extension (for marketplace).
+  # Zip extension configuration parameters changed on PHP 7.3
+  # see https://www.php.net/manual/en/zip.installation.php#zip.installation
+  && apt-get install --assume-yes --no-install-recommends --quiet libzip-dev zip \
+  && docker-php-ext-install zip \
+  \
+  # Enable apache mods.
+  && a2enmod rewrite \
+  \
+  # Install cron service.
+  && apt-get install --assume-yes --no-install-recommends --quiet cron \
+  \
+  # Install nodejs and npm.
+  && apt-get install --assume-yes --no-install-recommends --quiet gnupg \
+  && curl --silent --location https://deb.nodesource.com/setup_16.x | bash - \
+  && apt-get install --assume-yes --no-install-recommends --quiet nodejs \
+  \
+  # Install git and zip used by composer when fetching dependencies.
+  && apt-get install --assume-yes --no-install-recommends --quiet git unzip \
+  \
+  # Install gettext used for translation files.
+  && apt-get install --assume-yes --no-install-recommends --quiet gettext \
+  \
+  # Clean sources list
+  && rm -rf /var/lib/apt/lists/*
+
+# Copy composer binary
+COPY --from=composer /usr/bin/composer /usr/bin/composer
+
+# Copy files to container.
+COPY ./files/etc/apache2/sites-available/000-default.conf /etc/apache2/sites-available/000-default.conf
+COPY ./files/etc/cron.d/glpi /etc/cron.d/glpi
+COPY ./files/opt/startup.sh /opt/startup.sh
+COPY ./files/opt/download_glpi.sh /opt/download_glpi.sh
+COPY ./files/opt/install_glpi.sh /opt/install_glpi.sh
+
+# Install GLPI crontab.
+RUN crontab -u www-data /etc/cron.d/glpi
+
+WORKDIR /var/www/glpi
+
+# Download GLPI Files
+ARG GLPI_RELEASE
+RUN chmod u+x /opt/download_glpi.sh
+RUN /opt/download_glpi.sh ${GLPI_RELEASE}
```

Comment:
IMHO, the GLPI files should be directly included in images. It would permit to tag images according to contained GLPI version.
For instance, we could use following images names/tags :
 - GLPI 10.0.0 -> `glpi:10.0.0`, `glpi:10.0` (latest tag of `10.0` intermediate release),  `glpi:10` (latest tag of `10` major release), `glpi:latest`
 - GLPI 9.5.7 -> `glpi:9.5.7`, `glpi:9.5` (latest tag of `9.5` major release) ~~`glpi:9` (cannot consider that 9.1 and 9.5 are intermediate versions of a same major release)~~  
 - GLPI 9.5.6 -> `glpi:9.5.6`

So, people using `glpi` or `glpi:latest` will always get the latest stable release when using a `pull` command, but people using `glpi:10.0` would ensure to only get bugfixes versions of GLPI 10.0.

Also, if we let people choose the release to use in this image, we will have to maintain images that are compatible with multiple GLPI versions, which can be a pain.